app/navigation/routes.js

```diff
@@ -167,6 +177,14 @@ export const Routes = {
         navigationProps: {
             title: {id: 'mobile.routes.user_profile', defaultMessage: 'Profile'}
         }
+    },
+    Saml: {
+        key: 'Saml',
+        transition: RouteTransitions.Horizontal,
+        component: Saml,
+        navigationProps: {
+            title: {id: 'mobile.routes.saml', defaultMessage: 'Single SignOn'}
```

Comment:
Yeah, we might want to discuss how it'll work in the long term since right now we just have the login button text like "With SAML" in the config